src/phantom.js

```diff
@@ -45,7 +36,36 @@ export default class Phantom {
         }
 
         let pathToShim = path.normalize(__dirname + '/shim.js');
-        logger.debug(`Starting ${phantomPath} ${args.concat([pathToShim]).join(' ')}`);
+
+        let loggerLevel = process.env.DEBUG === 'true' ? 'debug' : 'info';
+        if(typeof config.logger === 'object') {
+            let isOk = true;
+            let checkList = ['debug', 'info', 'warn', 'error'];
+            for(let i = 0; i < checkList.length; i += 1) {
+                if(typeof config.logger[checkList[i]] !== 'function') {
+                    isOk = false;
+                    break;
+                }
+            }
+            if(isOk) {
+                this.logger = config.logger;
+            }
+        } else if(typeof config.logger === 'string') {
+            loggerLevel = config.logger;
+        }
+
+        if(!this.logger) {
+            this.logger = new winston.Logger({
+                transports: [
+                    new winston.transports.Console({
+                        level: loggerLevel,
+                        colorize: true
+                    })
+                ]
+            });
+        }
+
+        this.logger.debug(`Starting ${phantomPath} ${args.concat([pathToShim]).join(' ')}`);
```

Comment:
Do you still need all the code above? Ideally, we only change what is needed for this pull request.